examples/kubernetes/volume-expansion/README.md

```diff
@@ -0,0 +1,50 @@
+# Volume Expansion
+
+## Context
+
+Users are able to change the capacity of their volumes. 
+Currently, the capacity can ONLY be increased, and NOT decreased.
+Only storage capacity can be adjusted. Throughput can NOT be modified.
+See the [design document](../../../docs/design.md) for more information.
```

Comment:
no design document yet, also I'd add a disclaimer that scaling the capacity on an FSxZ volume increases the StorageCapacityQuota and StorageCapacityReservation to the same value (the requested storage)